include/nccl_ofi_mr.h

```diff
@@ -140,6 +145,16 @@ static inline void nccl_ofi_mr_ckey_fill_mr_attrs(nccl_ofi_mr_ckey_ref ckey, str
 	attrs->iov_count = 1;
 }
 
+static inline void nccl_ofi_mr_compute_page_address(uintptr_t addr,
+						    size_t size,
+						    uintptr_t system_page_size,
```

Comment:
no reason for this to be named system_page_size.  It's just a page size.

Reply:
Renamed the parameter.